### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- Added videos in db
- Added spinner
- Added filters based on category

### Preview
[Link](https://deploy-preview-5--always-stream.netlify.app/)

### CODE REVIEW

**Review by manojncsoni007 — APPROVED** [review]
looks good to me
- Nice UI
- clean code

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import { Explore, Home } from "./pages";
 
 function App() {
   return (
     <div className="App">
       <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/explore" element={<Explore />} />
       </Routes>
```
Comment (aniketsingh13) [inline src/App.js:13]: you can make seperate folder for this

**Review by aniketsingh13 — APPROVED** [review]
ui Looks great
code is also clean

File: src/App.js (hunk 1 above)
Comment (SanketDhabarde) [reply src/App.js:13]: Yes, I will at the end😅

**Pull request merged**